custom_components/kia_uvo/testAPI.py

```diff
@@ -0,0 +1,108 @@
+import logging
```

Comment:
do you want to keep this file here?

Reply:
Ideally yes.  It is very helpful for testing this new feature so thought I would save the effort from others.